plaso/output/shared_elastic.py

```diff
@@ -170,17 +171,19 @@ def _GetSanitizedEventValues(self, event, event_data, event_tag):
         event_values[attribute_name] = attribute_value
 
     # Add a string representation of the timestamp.
-    try:
-      attribute_value = timelib.Timestamp.RoundToSeconds(event.timestamp)
-    except TypeError as exception:
-      logger.warning((
-          'Unable to round timestamp {0!s}. error: {1!s}. '
-          'Defaulting to 0').format(event.timestamp, exception))
-      attribute_value = 0
-
-    attribute_value = timelib.Timestamp.CopyToIsoFormat(
-        attribute_value, timezone=self._output_mediator.timezone)
-    event_values['datetime'] = attribute_value
+    # TODO: determine why the string representation needs to be rounded in
```

Comment:
@berggren @kiddinn @Onager any idea why the string representation needs to be in seconds precision?

Reply:
This was introduced in https://github.com/log2timeline/plaso/commit/4d7b40ba21189d0f13c55435cf5e0a0e1a08bae0

Reply:
Discussed with @kiddinn and @berggren there is no good reason to keep this behavior, removing round to seconds behavior entirely.